src/main/webapp/script.js

```diff
@@ -105,6 +108,7 @@ function fetchAuthIndexQuestion() {
 
       const answerSubmission = document.getElementById('post-answer');
       if (answerSubmission) {
+        console.log('working');
```

Comment:
I think that this console logs should be deleted or commented.

Reply:
done